CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ In order to use automatic tests, we will store all our tests on `/tests` folder.
 We should call our test files with "test" before the description, like `test_whatYouWantToTest.py`. 
 Also the functions inside the test files should be called with "test" before the description, like `test_testFunctionName()`. That way `pytest` would know what files and function is our tests.
 
```

Comment:
no seeing changes to `CONTRIBUTING.md` mentions in the PR description - does this really belong here?

Reply:
I added it as a comment. Will edit the description as well.
It is necessary because we started using pipenv and the commands to run the tests are different now.